NMEA/index.html

```diff
@@ -13,7 +13,7 @@
 	<meta name="author" content="@sudheesh001"> <!-- CHANGE THIS -->
 	<link rel="icon" type="image/png" href="../../artwork/favicon.png">
 	<link href="../../css/bootstrap.min.css" rel="stylesheet">
-	<script src="../../js/jquery.js"></script>
+	<script src="../../js/jquery.min.js"></script>
```

Comment:
Doesn't look like the right path to me. Is this not throwing a `404` error in the browser console ?